ui/src/lib/access.worker.js

```diff
@@ -27,7 +24,7 @@ export async function generateAccess({
 	satelliteUrl = "12tRQrMTWUWwzwGh18i7Fqs67kmdhH9t6aToeiwbo5mfS2rUmo@us2.storj.io:7777",
 	passphrase = ""
 }) {
-	await instantiateStreaming;
+	await response;
```

Comment:
We have the same problem again. `generateAccess()` will wait for the fetch to succeed as `await response` is the the same as `await fetch(...)`. However, what we need to do is wait for the `instantiateStreaming()` to finish.

You can accomplish this easily by going from:

``` javascript
const response = fetch("/access.wasm");

instantiateStreaming(response, go.importObject)
	.then((result) => {
```

to

``` javascript
const response = instantiateStreaming(await fetch("/access.wasm"), go.importObject)
	.then((result) => {
```